docs/design/TEMPLATE.md

```diff
@@ -1,12 +1,8 @@
-<!--
-This is a template for TiDB's change proposal process, documented [here](./README.md).
--->
+# TiDB Design Documents
 
-# Proposal: <!-- Title -->
-
-- Author(s):     <!-- Author Name, Co-Author Name, with the link(s) of the GitHub profile page -->
-- Last updated:  <!-- Date -->
-- Discussion at: <!-- https://github.com/pingcap/tidb/issues/XXX -->
+- Author(s): @ghid1, @ghid2, ...
```

Comment:
I can tell what `ghid` mean, just wondering would it be better to explain more about this abbreviation.

Reply:
```suggestion
- Author(s): [Author Name](http://github.com/your-github-id), [Co-Author Name](http://github.com/your-github-id), ...
```

Reply:
@zz-jason Make sense, I use the previous form with expressive content.

Reply:
well I found the `@` link magic work and those tend to prevent `[]()` complexity